numpy/random/tests/test_generator_mt19937.py

```diff
@@ -755,6 +770,19 @@ def test_permutation(self):
         actual = random.permutation(arr_2d)
         assert_array_equal(actual, np.atleast_2d(desired).T)
 
+        # Test permutation along a different axis
```

Comment:
Could be a separate method `test_permutation_custom_axis` like https://github.com/numpy/numpy/blob/3deeea9522f5c512d52b3fe6d418d99b338eaece/numpy/random/tests/test_generator_mt19937.py#L586-L597